data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			`	"Velcome. I am called Hjlod." The thickly accented woman looks back from the driver's seat. She has short blonde hair, and her face and arms are covered in long weaving blue tattoos. "I am from here. I have spent whole life traversing this tundra. I know like back of hand." She gives you a lopsided grin and then pats the steering console. "And Brunhilda here, she never let you down."`
+			`	Hjlod turns away from you and starts pulling some levers. "Now," she says, "let's begin."`
+			choice
+				`	(Sit back as the motor hums and the Ice-Crawler lurches to a start.)`
+			`	After a few hours of crawling through a bleak, blank wasteland in silence, Hjlod turns the crawler into what looks like a bleaker and blanker wasteland.`
+			scene "scene/iceplains"
+			`	"This is famous Glaesisvellir ice-plain. Story say once whole convoy of researcher disappear here. Can drill hole in ice for fishing. There is very good tasting voolly octopus beneath it." Hjlod slows down and looks back at you. "You have question?"`
+			label questions
+			choice
+				`	"Nope. No questions."`
+					goto noquestion
+				`	"What happened to the researchers?"`
+				`	"Is it safe to drive over the ice?"`
+					goto ice
+				`	"Are there sea-monsters beneath the ice?"`
+					goto monster
+			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
+			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book dumb."`
+			`	"Any more question?"`
+				goto questions
+			label ice
+			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
+			` 	"You have more question?"`
+				goto questions
+			label monster
+			`	"Every world have animal. Some animal big, some small. Voolly octopus, mostly small. Size of this Ice-Crawler, only. Vorst in Glaesisvellir ice-plains is Ice-Slug. They can squeeze through tiniest hole in ice. This take long time, for they are as big as house. They have no teeth, too hard to squeeze through little hole, but they have acid spit. Put hole in Ice-Crawler, easy. They eat many foolish researcher and tourist."`
+			`	She smiles at you. "You're not foolish tourist. You are with Hjlod. You are safe."`
+			`	"Any more question?"`
+				goto questions
+			label noquestion
+			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit. "Enough ice-plain, now ve go to next stop."`
```

Comment:
This response will still appear even if the player has asked questions, which makes the `Is smart, not to ask question` bit feel a little weird. Consider rephrasing this line to avoid that issue.

Reply:
I was thinking that, in the event you asked questions, Hjlod here is commenting on the fact it is smart not to ask anymore questions.